lib/views/help/requesting.es.html.erb

```diff
@@ -477,22 +474,23 @@
         <strong>El Habeas Data</strong> garantiza la protección de los datos privados y el acceso de toda persona a cualquier información que pueda tener cualquier entidad pública sobre ella, así como el derecho a saber por qué y con qué finalidad tiene esa información. ( Ver Punto 6)
       </p>
       <h2>
```

Comment:
Ah, this needed an `id` attribute so that the anchors have a target:

```diff
- <h2>
+ <h2 id="recursos">
```

I'll do this when I merge it in